MacOSX/scripts/postinstall

```diff
@@ -15,8 +15,18 @@ else
 	mv /Library/OpenSC/etc/opensc.conf.orig /Library/OpenSC/etc/opensc.conf
 	md5 -r /Library/OpenSC/etc/opensc.conf  > /Library/OpenSC/etc/opensc.conf.md5
 fi
+
 for f in /Library/OpenSC/bin/*
 do
 	ln -sf $f /usr/local/bin
 done
+
+for f in /Library/LaunchAgents/pkcs11-register.plist /Library/LaunchAgents/pkcs11-register.plist
```

Comment:
Twice the same in loop ? "/Library/LaunchAgents/pkcs11-register.plist"

Reply:
copy/paste error. thanks, fixed.